### PR TITLE
Refresh update

### PR DESCRIPTION
**Pull request by mqq-marek**
There are few small changes I do in a way to make benchmark running on my site.
Please use them on your License terms if you are ok with them.

### CODE REVIEW

**Comment by MartinThoma** [comment]
Why did you delete the files?

File: benchmark.py [hunk 1]
```diff
@@ -48,8 +49,11 @@ def main(
 ) -> None:
     cache_path = Path("cache.json")
     if cache_path.exists():
-        with open(cache_path) as f:
-            cache = Cache.parse_obj(json.load(f))
+        try:
+            with open(cache_path) as f:
+                cache = Cache.model_validate(json.load(f))
+        except JSONDecodeError:
+            cache = Cache()
```
Comment (MartinThoma) [inline benchmark.py:56]: Nice!

**Comment by MartinThoma** [comment]
I like your changes, but I don't understand why you deleted so many files + added them to .gitignore. Those files are under version control on purpose. I want to be able to see and compare the changes in text  extraction over different versions of the libraries.

**Comment by mqq-marek** [comment]
Hi Martin,After doing this I noticed that it can by wrong from goal of this project.My workflow is that I never keep in repo thing which are generated by code.Here you want to present resalt than my approach is bad,Other point is, when you want to present results why do not keep pdfs in repo. but only ground truth, that is the same type of question. what about when source pdfs and will by unavailable or changed.Anyway it is my first approach to contribute something to open source.If you are ok with my changes i can make another pull request without this changes.btw. reason i use this code i test text-extraction on other set of pdfs and do few others changes like adding config using dynaconf with ability to define which file and which extract method will be processed during benchmark run.thanks MarekSent from my iPadOn 29 Oct 2023, at 04:56, Martin Thoma ***@***.***> wrote:﻿
I like your changes, but I don't understand why you deleted so many files + added them to .gitignore. Those files are under version control on purpose. I want to be able to see and compare the changes in text  extraction over different versions of the libraries.

**Comment by MartinThoma** [comment]
> My workflow is that I never keep in repo thing which are generated by code

In almost all cases, that is a good approach! Typically, generated files are not interesting, but only exist to be used (e.g. minified Javascript, compiled binaries).

However, in this repository we don't want to run an application. We want to see/present the results. The results are the core part of this repository.

The code and the input documents are here only for two reasons:

1. **Reproducibility**: I want people to be able to run the scripts on their own. The numbers (speed) will be slightly different, but overall it should look similar.
2. **Updates**: I update the benchmark results from time to time. I also hope that at some point others will contribute to this repository / enhance the results (e.g. add libraries I forgot or a new benchmark category). I realize that this might be hard as I didn't structure it well


> Anyway it is my first approach to contribute something to open source. If you are ok with my changes i can make another pull request without this changes.

Well done! Yes, if you undo the deletions, I'll happily merge the rest :-) (Feel free to just create a new PR, but you could also update this PR via `git checkout master -- <filename/name of the folder>`

> i use this code i test text-extraction on other set of pdfs

Interesting! If you have some more insights, please let me know!

One thing I noticed is that issues in whitespace-extraction are not represented well by the current metric. However, I don't know how to improve the metric to represent the quality in a better way.

**Comment by mqq-marek** [comment]
I make another branch without deleting files and create new pull request.

**Comment by MartinThoma** [comment]
I'm closing this PR in favor of https://github.com/py-pdf/benchmarks/pull/10

**Pull request closed without merging**